settings.gradle

```diff
@@ -1,6 +1,6 @@
 include ':sdk:communication:azure-communication-chat'
-include ":sdk:core:azure-core"
-include ":sdk:storage:azure-storage-blob"
-include ":sdk:communication:azure-communication-common"
-include ":sdk:template:azure-sdk-template"
-include ":samples:sample-app-storage"
+include ':sdk:communication:azure-communication-common'
+include ':sdk:core:azure-core'
+include ':sdk:storage:azure-storage-blob'
+include ':sdk:template:azure-sdk-template'
+include ':samples:sample-app-storage'
```

Comment:
Are we planning on using single quotes across our Gradle files?

Reply:
Generally I follow the advice of Ruby which also has this distinction (single quotes are literals, double quotes allow interpolation) and use single quotes everywhere I don't intend for interpolation to occur. I only changed them here because I was already re-ordering them. I considered changing them everywhere but felt that was too disruptive for this PR.